src/main/java/org/folio/service/CirculationSettingsService.java

```diff
@@ -67,4 +71,29 @@ public Future<Response> delete(String circulationSettingsId) {
       .compose(eventPublisher.publishRemoved(circulationSetting))
     );
   }
+
+  private Future<CirculationSetting> updateSettingsValue(CirculationSetting circulationSetting, Throwable throwable) {
+    if (!isDuplicate(throwable.getMessage())) {
+      return Future.failedFuture(throwable);
+    }
+
+    return getSettingsByName(circulationSetting.getName())
+      .compose(settings -> updateSettings(settings, circulationSetting));
+  }
+
+  private Future<CirculationSetting> updateSettings(List<CirculationSetting> settings, CirculationSetting circulationSetting) {
```

Comment:
The line must not exceed 100 characters in length

Reply:
done